src/ui/camera.js

```diff
@@ -857,6 +857,18 @@ class Camera extends Evented {
      * @see [Fly to a location based on scroll position](https://www.mapbox.com/mapbox-gl-js/example/scroll-fly-to/)
      */
     flyTo(options: Object, eventData?: Object) {
+        // Fall throwugh to jumpTo is user has set prefers-reduced-motion
+        if (browser.prefersReducedMotion) {
+            this.jumpTo({
+                center: options.center,
+                zoom: options.zoom,
+                bearing: options.bearing,
+                pitch: options.pitch,
+                around: options.around
+            }, eventData);
+            return;
```

Comment:
Should be `return this` (or `return this.jumpTo(...)`) for consistency/chaining.